packages/components/src/badge/index.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import { ReactNode } from 'react';
```

Comment:
Nit: this should be a type import.
```suggestion
import type { ReactNode } from 'react';
```